README.md

```diff
@@ -18,7 +20,9 @@ This is the backend for Votte, which will allow Flushing Tech members to vote on
 
 All data is stored in a PostgreSQL database, ensuring reliable data management and integrity.
 
-Spin it up today with `docker-compose up -d`
+Spin it up today with `docker compose up -d`
+
+Rename `.env.template` to `.env`. The `DATABASE_URL` is configured to connect to the Docker PostgreSQL database, while the remaining values are _mock placeholders_ to enable spinning up the Node server. If you want to test the authentication flow, you must provide your own values.
```

Comment:
very helpful documentation, thanks!

you can also include a link to the google oauth instructions: https://support.google.com/cloud/answer/6158849?hl=en

Reply:
Got you. Link this to #22